tests/functional/test_data_classes.py

```diff
@@ -1138,6 +1139,30 @@ def test_base_proxy_event_json_body_with_base64_encoded_data():
     assert event.json_body == data
 
 
+def test_kafka_event():
+    event = KafkaEvent(load_event("kafkaEvent.json"))
+    assert event.event_source == "aws:SelfManagedKafka"
```

Comment:
Could you please change the event source to "aws:kafka" and add the eventSourceArn field?